### PR TITLE
Upgrade iStumbler.app to v101

### PR DESCRIPTION
**Pull request by ghost**
Upgrade iStumbler.app to v101

### CODE REVIEW

File: Casks/istumbler.rb [hunk 1]
```diff
@@ -4,10 +4,14 @@
     version '99'
     sha256 'ac30e44fe86132c93a5b33699de00e86628e2f51a42015d2225b91521a198b63'
     url "http://istumbler.net/archive/release#{version}/downloads/iStumbler-#{version}.zip"
-  else
+  elsif MacOS.release == :mavericks
     version '100'
     sha256 '71f6a6b0e255a853664ed4900835a42f2d23dcb05de35acfb3ac2ec1c5fb2edc'
     url "https://istumbler.net/downloads/istumbler-#{version}.dmg"
+  elsif MacOS.release == :yosemite
```
Comment (vitorgalvao) [inline Casks/istumbler.rb]: Should be just `else`.

File: Casks/istumbler.rb [hunk 2]
```diff
@@ -15,4 +19,4 @@
   license :unknown    # todo: change license and remove this comment; ':unknown' is a machine-generated placeholder
 
   app 'iStumbler.app'
-end
+end
```
Comment (vitorgalvao) [inline Casks/istumbler.rb]: It has a “no newline at end of file” warning. Those are [actually meaningful](http://unix.stackexchange.com/a/18789).

**Comment by mahuani** [comment]
Changes made

**Comment by vitorgalvao** [comment]
Thank you, could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by mahuani** [comment]
Squashed

**Pull request merged**